lib/gemstone/infomon/parser.rb

```diff
@@ -412,7 +415,27 @@ def self.parse(line)
               else
                 :noop
               end
-            when Pattern::HouseCHE
+            when Pattern::ProfileStart
+              @profile_self_output = true
```

Comment:
are we doing this in a lot of places? We should probably rewrite this to be `@operation = <some enum>`